plugins/module_utils/copy.py

```diff
@@ -216,6 +228,87 @@ def copy_vsam_ps(src, dest):
     return rc, out, err
 
 
+def copy_asa_uss2mvs(src, dest):
+    """Copy a file from USS to an ASA sequential data set or PDS/E member.
+
+    Arguments:
+        src: {str} -- Path of the USS file
+        dest: {str} -- The MVS destination data set or member
+
+    Returns:
+        boolean -- The return code after the copy command executed successfully
+        str -- The stdout after the copy command executed successfully
+        str -- The stderr after the copy command executed successfully
+    """
+    module = AnsibleModuleHelper(argument_spec={})
+
+    oget_cmd = 'tsocmd "OGET \'{0}\' \'{1}\'"'.format(src, dest)
```

Comment:
This is not a real issue, back in the beginning we had issues with `tsocmd` , likely from the Rocket Python z port, can't recall the issue. I would like to offline discuss the use of OGET also wondering if you thought there was any value in using module_utils -> mvs_cmd -> ikjeft01(...) to invoke OGET, it drives mvscmd which can also have limitations with temporary data sets, no action here but just wanted to have a discussion.

Reply:
Could change `tsocmd` for `mvscmd` without too much problem

Reply:
Looks good to me, pending pipeline testing I approve this and thank you for the change.